client/src/components/Intro/index.js

```diff
@@ -57,6 +59,13 @@ function Intro({
           <Link className='btn btn-lg btn-primary btn-block' to='/settings'>
             Update my account settings
           </Link>
+          {completedChallengeCount > 1 ? (
```

Comment:
```suggestion
          {completedChallengeCount > 0 ? (
```
This should be available as soon as a user has completed a challenge.

Reply:
Fixed it